### PR TITLE
APR-1051, APR-1052 and APR-1054 - section 7.7 – Policy for profession…

### PR DESCRIPTION
**Pull request by MakCom**
Work is completed for tickets APR-1051, APR-1052 and APR-1054, which is section 7.7 – “Policy for professional development of employees”.

### CODE REVIEW

File: src/SFA.DAS.QnA.Database/projects/roatp/sections/section7.7.json [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+                    "ReturnId": "7710",
+                    "Conditions": [],
+                    "ConditionMet": false
+                }
+            ],
+            "Complete": false,
+            "AllowMultipleAnswers": false,
+            "Order": null,
+            "Active": true,
+            "NotRequiredConditions": [
```
Comment (David-B-Read) [inline src/SFA.DAS.QnA.Database/projects/roatp/sections/section7.7.json]: Can this empty array be removed? If it's generated by the QnA tool, not a problem

File: src/SFA.DAS.QnA.Database/projects/roatp/sections/section7.7.json [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+                    "ReturnId": "7.7",
+                    "Conditions": [],
+                    "ConditionMet": false
+                }
+            ],
+            "Complete": false,
+            "AllowMultipleAnswers": false,
+            "Order": null,
+            "Active": false,
+            "NotRequiredConditions": [
```
Comment (David-B-Read) [inline src/SFA.DAS.QnA.Database/projects/roatp/sections/section7.7.json]: As per comment on page 7700

File: src/SFA.DAS.QnA.Database/projects/roatp/sections/section7.7.json (hunk 1 above)
Comment (MakCom) [reply src/SFA.DAS.QnA.Database/projects/roatp/sections/section7.7.json]: @David-B-Read Yes, it’s generated by QnA Config Tool. I also checked some other section’s json files and they all have empty arrays, so to be consistent I will leave it as that.

File: src/SFA.DAS.QnA.Database/projects/roatp/sections/section7.7.json (hunk 2 above)
Comment (MakCom) [reply src/SFA.DAS.QnA.Database/projects/roatp/sections/section7.7.json]: answer as per page 7700

**Pull request merged**